lambdas/src/common/security/jwt-verifier.ts

```diff
@@ -95,8 +120,7 @@ export class JwtVerifier {
     }
 
     public clearJWKSCache() {
-        cachedJWKS = null;
-        cachedJWKSExpiry = null;
+        cachedJWKS = {};
```

Comment:
Do we definitely want to clear the whole cache or do we only want to clear the cache for that endpoint?